README.md

```diff
@@ -4,6 +4,12 @@
 module "jinad" {
    source         = "jina-ai/jinad-aws/jina"
    version        = "0.0.5"
+   ebs = {
+     device_name = "/dev/sdh"
+     device_name_renamed = "/dev/xvdh"
+     mount_location = "/mnt/data"
+     jina_home = "/usr/local/jina"
+     }
```

Comment:
```hcl
   ebs = {
     device_name = "/dev/sdh"
     device_name_renamed = "/dev/xvdh"
     mount_location = "/mnt/data"
     jina_home = "/usr/local/jina"
}
```

Is this mandatory for the user to pass? If it is not, let's remove it from docs, as it is confusing.

Reply:
Fine !